src/sql/platform/connection/common/connectionManagement.ts

```diff
@@ -212,6 +212,11 @@ export interface IConnectionManagementService {
 
 	getUniqueConnectionProvidersByNameMap(providerNameToDisplayNameMap: { [providerDisplayName: string]: string }): { [providerDisplayName: string]: string };
 
+	/**
+	 * Gets the default authentication type from the configuration service
+	 */
+	getDefaultAuthenticationTypeId(): string;
```

Comment:
This is the newly added method. You'll find references to it through various interfaces and classes.